fetch.bs

```diff
@@ -2393,7 +2393,7 @@ with a <i>CORS flag</i> and <i>recursive flag</i>, run these steps:
  <a lt="block bad port">should fetching <var>request</var> be blocked due to a bad port</a>,
  <a href=https://w3c.github.io/webappsec-mixed-content/#should-block-fetch>should fetching <var>request</var> be blocked as mixed content</a>,
  or
- <a href=https://w3c.github.io/webappsec-csp/#should-block-request>should fetching <var>request</var> be blocked by Content Security Policy</a>
+ [[csp-3#should-block-request]]
```

Comment:
Yes, if you tag a heading with one of the dfn-type attributes (like `dfn`, here) it becomes a definition too.  (And as Anne's example shows, if you're using `dfn`, don't forget to `export` it too unless you want it private.)